config/default/core.entity_view_mode.paragraph.hero_with_overlay.yml

```diff
@@ -0,0 +1,10 @@
+uuid: 90e29480-6f84-4ed0-a1ff-d27964d226ca
```

Comment:
Another view mode for all of the paragraphs that only one will use?

We had this problem with the stanford_paragraphs module and went with a generic naming convention so that each P-Type could use each view mode for something and have it be slightly more meaningful.

Reply:
yeah i was on the fence. :person_fencing:  the nice thing is that a new entity_view_display config won't be created unless that view mode is enabled for another p-type.